test/e2e/daemonset_test.go

```diff
@@ -196,6 +196,7 @@ func getVertxDeployment(namespace string, selector map[string]string) *appsv1.De
 								},
 							},
 							InitialDelaySeconds: 1,
+							TimeoutSeconds:      3,
```

Comment:
@madeelrh what is the default timeout before? Does it cause any issue?

Reply:
Default is 1 second. The Vertx code has a delay of 1 second. With the delay in place, the Liveness service fails with 1 second default timeout. 
The delay can be found here:
 https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_jaegertracing_vertx-2Dcreate-2Dspan_blob_7014795d0d59f22c218b5dd4edc6d9bda0b9496b_src_main_java_com_example_demo_MainVerticle.java-23L92&d=DwIDaQ&c=jf_iaSHvJObTbx-siA1ZOg&r=1FWlI4SNjakgGk6Vgm0TFwPV1aews_uxyG3WGENXUtg&m=fO0TkOxFlw65Sa8EPsFKFcpHAS14M3cuwEbNKXSWb9c&s=936v1QacYoguKnSpJo4N1ftn4djTxaHIy2uVETI-Osc&e=

Reply:
The `InitialDelaySeconds` is already accounting for the 1s delay. There should not be any need for a `TimeoutSeconds` here because of that.